modules/oke/nodepools.tf

```
@@ -64,6 +64,12 @@ resource "oci_containerengine_node_pool" "nodepools" {
     defined_tags  = merge(lookup(var.defined_tags, "node", {}), lookup(each.value, "node_defined_tags", {}))
   }
 
+  node_eviction_node_pool_settings {
+    #Optional
+    eviction_grace_duration              = format("PT%sM",lookup(each.value, "eviction_grace_duration", 0))
+    is_force_delete_after_grace_duration = lookup(each.value, "force_node_delete", true)
```

```suggestion
    is_force_delete_after_grace_duration = tobool(lookup(each.value, "force_node_delete", true))
```
Worth wrapping in `tobool()` to be extra safe when pulling out of an untyped map, since Terraform will convert types.